config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  root 'messages#index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
```

Comment:
As said in the Front end repository, kindly check that you are correctly connecting both repositories via the correct routes. To help you do this you can check your back-end console and the network end-point response in your developer web tools in your browser.

![image](https://github.com/Luchy-B/hello_rails_back_end/assets/93630700/d6410551-7f3f-47cf-8696-d12104bd268a)

![image](https://github.com/Luchy-B/hello_rails_back_end/assets/93630700/750bebc0-5ef8-4c34-b303-2eb2b7658150)